build.gradle

```diff
@@ -12,12 +12,9 @@ buildscript {
 
 allprojects {
     group = 'com.github.ytai.ioio'
-    version = 5.07
+    version = getTag()
```

Comment:
Where is getTag() defined? I'm not up to speed on how this is generated. Do we need to tag every commit or this is generated via the commit number automatically by the code you added a while ago?

Reply:
It comes from here https://github.com/ytai/ioio/blob/master/build.gradle#L38-L56
You only create a tag and you are done